<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Almacen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="contMDI"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionAbrir_sesi_n"/>
    <addaction name="actionCerrar_sesi_n"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menu_Opciones">
    <property name="title">
     <string>O&amp;pciones</string>
    </property>
    <widget class="QMenu" name="menuCambiar_idioma">
     <property name="title">
      <string>Cambiar idioma</string>
     </property>
    </widget>
    <addaction name="action_Configuracion"/>
    <addaction name="actionCambiar_Contrase_a"/>
    <addaction name="menuCambiar_idioma"/>
   </widget>
   <widget class="QMenu" name="menuVEr">
    <property name="title">
     <string>Ver</string>
    </property>
    <widget class="QMenu" name="menuVentanas">
     <property name="title">
      <string>Ventanas</string>
     </property>
     <addaction name="actionCascada_2"/>
     <addaction name="actionMosaico_2"/>
     <addaction name="separator"/>
     <addaction name="actionCerrar_todas"/>
    </widget>
    <widget class="QMenu" name="menuInformes">
     <property name="title">
      <string>Informes</string>
     </property>
     <widget class="QMenu" name="menuListados">
      <property name="title">
       <string>Listados</string>
      </property>
      <widget class="QMenu" name="menuProductos">
       <property name="title">
        <string>Productos</string>
       </property>
       <addaction name="actionProductis"/>
       <addaction name="actionCategor_as"/>
       <addaction name="actionFamilias_2"/>
       <addaction name="actionHist_rico_de_certificaciones"/>
      </widget>
      <widget class="QMenu" name="menuMovimientos">
       <property name="title">
        <string>Movimientos</string>
       </property>
       <addaction name="actionSalidas"/>
       <addaction name="actionEntradas"/>
       <addaction name="actionEnvios"/>
       <addaction name="actionRecepciones"/>
       <addaction name="actionPedidos"/>
       <addaction name="actionStock"/>
      </widget>
      <widget class="QMenu" name="menuOperarios">
       <property name="title">
        <string>Operarios</string>
       </property>
       <addaction name="actionOperarios_3"/>
       <addaction name="actionPartes_de_Trabajo"/>
      </widget>
      <addaction name="menuMovimientos"/>
      <addaction name="actionKits_2"/>
      <addaction name="menuProductos"/>
      <addaction name="actionVeh_culos"/>
      <addaction name="menuOperarios"/>
      <addaction name="actionCentros_2"/>
     </widget>
     <addaction name="menuListados"/>
     <addaction name="actionProductos_por_operario"/>
     <addaction name="actionEnv_os_creados_a_partir_de_pedidos"/>
    </widget>
    <addaction name="menuVentanas"/>
    <addaction name="separator"/>
    <addaction name="menuInformes"/>
    <addaction name="actionEstad_sticas"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuVEr"/>
   <addaction name="menu_Opciones"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="tbarGeneral">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMovimientos"/>
   <addaction name="actionKits"/>
   <addaction name="actionProductos"/>
   <addaction name="actionVehiculos"/>
   <addaction name="actionOperarios"/>
   <addaction name="actionCentros"/>
   <addaction name="separator"/>
   <addaction name="actionSalir_Menu"/>
  </widget>
  <action name="actionProductos">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/iconos/iconoProductos.png</normaloff>:/iconos/iconos/iconoProductos.png</iconset>
   </property>
   <property name="text">
    <string>Productos</string>
   </property>
   <property name="shortcut">
    <string>Alt+H</string>
   </property>
  </action>
  <action name="actionVehiculos">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/Vehiculos/iconos/iconoVEHICULO.png</normaloff>:/iconos/Vehiculos/iconos/iconoVEHICULO.png</iconset>
   </property>
   <property name="text">
    <string>Vehiculos</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="actionOperarios">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/Operarios/iconos/iconoOperario.jpg</normaloff>:/iconos/Operarios/iconos/iconoOperario.jpg</iconset>
   </property>
   <property name="text">
    <string>Operarios</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionCentros">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/Centros/iconos/iconoCENTROS.jpg</normaloff>:/iconos/Centros/iconos/iconoCENTROS.jpg</iconset>
   </property>
   <property name="text">
    <string>Centros</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionCategorias">
   <property name="text">
    <string>Categorías</string>
   </property>
   <property name="toolTip">
    <string>Categorias</string>
   </property>
  </action>
  <action name="actionProductos_Lista">
   <property name="icon">
    <iconset>
     <normaloff>:/iconos/Herramientas/iconos/iconoHERRAMIENTALista.png</normaloff>:/iconos/Herramientas/iconos/iconoHERRAMIENTALista.png</iconset>
   </property>
   <property name="text">
    <string>Lista</string>
   </property>
  </action>
  <action name="actionVehiculos_Lista">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/Vehiculos/iconos/iconoVEHICULOLista.png</normaloff>:/iconos/Vehiculos/iconos/iconoVEHICULOLista.png</iconset>
   </property>
   <property name="text">
    <string>Lista</string>
   </property>
  </action>
  <action name="actionCentros_Lista">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/Centros/iconos/iconoCENTROSLista.jpg</normaloff>:/iconos/Centros/iconos/iconoCENTROSLista.jpg</iconset>
   </property>
   <property name="text">
    <string>Lista</string>
   </property>
  </action>
  <action name="actionOperarios_Lista">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/Operarios/iconos/iconoOperarioLista.jpg</normaloff>:/iconos/Operarios/iconos/iconoOperarioLista.jpg</iconset>
   </property>
   <property name="text">
    <string>Lista</string>
   </property>
  </action>
  <action name="actionHistorico">
   <property name="text">
    <string>Histórico</string>
   </property>
   <property name="toolTip">
    <string>Historico</string>
   </property>
  </action>
  <action name="actionAbrir_sesi_n">
   <property name="text">
    <string>A&amp;brir sesión</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionCerrar_sesi_n">
   <property name="text">
    <string>&amp;Cerrar sesión</string>
   </property>
  </action>
  <action name="action_Configuracion">
   <property name="text">
    <string>Co&amp;nfiguración</string>
   </property>
  </action>
  <action name="actionOperarios_Partes_de_trabajo">
   <property name="text">
    <string>Partes de trabajo</string>
   </property>
  </action>
  <action name="actionSalir_Menu">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/iconos/iconoSalir.png</normaloff>:/iconos/iconos/iconoSalir.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionCambiar_Contrase_a">
   <property name="text">
    <string>Cambiar contraseña</string>
   </property>
  </action>
  <action name="actionMovimientos">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/iconos/iconoMOVIMIENTOS.jpg</normaloff>:/iconos/iconos/iconoMOVIMIENTOS.jpg</iconset>
   </property>
   <property name="text">
    <string>Movimientos</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionKits">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/iconos/iconos/iconoKIT.jpg</normaloff>:/iconos/iconos/iconoKIT.jpg</iconset>
   </property>
   <property name="text">
    <string>Kits</string>
   </property>
   <property name="shortcut">
    <string>Alt+K</string>
   </property>
  </action>
  <action name="actionFamilias">
   <property name="text">
    <string>Familias</string>
   </property>
  </action>
  <action name="actionMosaico">
   <property name="text">
    <string>Mosaico</string>
   </property>
  </action>
  <action name="actionCascada_2">
   <property name="text">
    <string>Cascada</string>
   </property>
  </action>
  <action name="actionMosaico_2">
   <property name="text">
    <string>Mosaico</string>
   </property>
  </action>
  <action name="actionCerrar_todas">
   <property name="text">
    <string>Cerrar todas</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionEstad_sticas">
   <property name="text">
    <string>Estadísticas</string>
   </property>
  </action>
  <action name="actionKits_2">
   <property name="text">
    <string>Kits</string>
   </property>
  </action>
  <action name="actionVeh_culos">
   <property name="text">
    <string>Vehículos</string>
   </property>
  </action>
  <action name="actionCentros_2">
   <property name="text">
    <string>Centros</string>
   </property>
  </action>
  <action name="actionSalidas">
   <property name="text">
    <string>Salidas</string>
   </property>
  </action>
  <action name="actionEntradas">
   <property name="text">
    <string>Entradas</string>
   </property>
  </action>
  <action name="actionEnvios">
   <property name="text">
    <string>Envíos</string>
   </property>
  </action>
  <action name="actionRecepciones">
   <property name="text">
    <string>Recepciones</string>
   </property>
  </action>
  <action name="actionPedidos">
   <property name="text">
    <string>Pedidos</string>
   </property>
  </action>
  <action name="actionStock">
   <property name="text">
    <string>Stock</string>
   </property>
  </action>
  <action name="actionProductis">
   <property name="text">
    <string>Productos</string>
   </property>
  </action>
  <action name="actionCategor_as">
   <property name="text">
    <string>Categorías</string>
   </property>
  </action>
  <action name="actionFamilias_2">
   <property name="text">
    <string>Familias</string>
   </property>
  </action>
  <action name="actionHist_rico_de_certificaciones">
   <property name="text">
    <string>Histórico de certificaciones</string>
   </property>
  </action>
  <action name="actionOperarios_3">
   <property name="text">
    <string>Operarios</string>
   </property>
  </action>
  <action name="actionPartes_de_Trabajo">
   <property name="text">
    <string>Partes de Trabajo</string>
   </property>
  </action>
  <action name="actionProductos_por_operario">
   <property name="text">
    <string>Productos limitados por operario</string>
   </property>
  </action>
  <action name="actionEnv_os_creados_a_partir_de_pedidos">
   <property name="text">
    <string>Envíos creados a partir de pedidos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
